api/client/samples/analogous_years/readme.md

```diff
@@ -0,0 +1,158 @@
+# Analogous Years
+Users can find similarity based ranks between a time period and a list of time periods 
+given by the time period ± 1 year, ± 2 years, ..., ± N years using `Analogous Years` application.
+For rank computation a user must provide:
+
+1. A time period determined by `initial_date` and `final_date` such that they are within a year of
+each other.
+2. Gro-entity(ies) and corresponding weights to compare the time periods.
+
+## Working Example Using Python
+To get started with `Analogous Years`, users have to install `Gro API Client`
+as detailed [here](https://github.com/gro-intelligence/api-client). 
+
+If the user wants to know which period of time is closest to the time period 
+between 1<sup>st</sup> January 2019 and 31<sup>st</sup> October 2019 with respect to the
+following Gro-entities -
+1. Rainfall (metric_id=2100031, item_id=2039, source_id=35, frequency_id=1)
```

Comment:
We actually have multiple different rainfall sources, so it might be a bit misleading to say this represents rainfall. Source would need to be specified in the name, as well.

My inner Nemo voice also has an aversion to setting ID numbers. But it's probably fine here.

Reply:
Added the name of the sources.